addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5247,7 +5247,7 @@ msgstr ""
 #: xbmc\video\dialogs\GUIDialogAudioSubtitleSettings.cpp
 #: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
 msgctxt "#12376"
-msgid "Set as default for all videos"
+msgid "Set as default for all files"
```

Comment:
well, streamed media is not a "file" - that's why it looked a bit misplaced to me and I suggested the shorter version. Still need to test ADSP to get my head around it's functionality